### PR TITLE
RESTEasy Reactive: treat multipart file input as regular input

### PR DESCRIPTION
**Pull request by FroMage**
If the user doesn't care about the file name, treat it like a normal value

### CODE REVIEW

File: independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/multipart/MultipartSupport.java [hunk 1]
```diff
@@ -55,27 +56,59 @@ public static MultipartFormDataInput toMultipartFormDataInput(ResteasyReactiveRe
         return formData.toMultipartFormDataInput();
     }
 
-    @SuppressWarnings({ "unchecked", "rawtypes" })
-    public static Object convertFormAttribute(String value, Class type, Type genericType, MediaType mediaType,
-            ResteasyReactiveRequestContext context, String attributeName) {
+    public static Object getConvertedFormAttribute(String attributeName, Class type, Type genericType, MediaType mediaType,
```
Comment (geoand) [inline independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/multipart/MultipartSupport.java:59]: Let's please remove lambdas from the code

**Review by geoand — COMMENTED** [review]
I added a small comment

File: independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/multipart/MultipartSupport.java (hunk 1 above)
Comment (FroMage) [reply independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/multipart/MultipartSupport.java:74]: Yup, fair. I found 40+ implementations of a constant `Supplier` in our code base but none of them directly accessible :(

**Comment by FroMage** [comment]
Rewritten without lambda.

**Review by geoand — APPROVED** [review]
Thanks!

**Comment by FroMage** [comment]
Re-running the JDK11 job, for a faliure that doesn't seem related, and didn't fail for JDK17:
```
2023-03-02T16:26:07.7139066Z [ERROR]   ExceptionInWriterTest.test:27 » SocketTimeout Read timed out
```

**Pull request merged**